### ISSUE TITLE
Delete conversation when done?

### ISSUE THREAD

**Issue by luanon404** [dsdanielpark/Bard-API#184]
Thank you.

**Comment by dsdanielpark** [comment]
I'm not sure in what situation you would need to delete a conversation by code line, but while it's not impossible, I believe it might be rare relative to the development effort and time involved. Have you considered implementing a simpler solution on the backend, such as initializing variables, to achieve this? If you have a specific solution in mind, please feel free to modify it and submit a Pull Request; we would appreciate it.

**Comment by luanon404** [comment]
> I'm not sure in what situation you would need to delete a conversation by code line, but while it's not impossible, I believe it might be rare relative to the development effort and time involved. Have you considered implementing a simpler solution on the backend, such as initializing variables, to achieve this? If you have a specific solution in mind, please feel free to modify it and submit a Pull Request; we would appreciate it.

because when upload image, you created new conversation instead of use current, i know its very hard to parse output, so i just want delete conversation after use instead of go https://myactivity.google.com/product/bard?utm_source=bard&pli=1 and delete. btw do you think proxy can fix 409 error code?

**Comment by luanon404** [comment]
i think i made the requests too fast, maybe 30 seconds per request seems ok
